data/dwarfplanets.ssc

```diff
@@ -37,8 +37,6 @@ ReferencePoint "Pluto-Charon" "Sol"
 {
     CustomOrbit "pluto"
 
-    OrbitColor  [ 0.556863 0.235294 0.576471 ]
```

Comment:
why did you remove this color?

Reply:
Since the orbit color for dwarf planets is now specified in the source code, that line is no longer needed

Reply:
Added it back (I can use rebase now!)